less/custom.less

```diff
@@ -59,4 +59,92 @@
 
 #header-row{
   display:none;
-}
\ No newline at end of file
+
+}
+
+.btn-gold {
+  text-transform: uppercase;
+  font-weight: 400;
+  font-size: 18px;
+  color: #feb322;
+  border: 3px solid #feb322;
```

Comment:
Can you add the focus, visited and hover here like that:

&:hover, &:focus, &:visited {
  color: #eda822;
  border-color: #eda822;
}